pkg/kinflate/app/application.go

```diff
@@ -39,8 +45,67 @@ func New(loader loader.Loader) (Application, error) {
 }
 
 // Resources computes and returns the resources from the manifest.
-func (a *applicationImpl) Resources() []resource.Resource {
+func (a *applicationImpl) Resources() ([]*resource.Resource, error) {
 	// This is where all the fun happens.
 	// From the manifest create the resources.
-	return nil
+	manifestBytes, err := a.loader.Load(constants.KubeManifestFileName)
+	if err != nil {
+		return nil, err
+	}
+
+	var m v1alpha1.Manifest
+	err = yaml.Unmarshal(manifestBytes, &m)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
May be combine the above code in a method on application called loadManifest() ?